### PR TITLE
🔧 Refactor manual-install Compose.yml: Simplify Environment Variables

### PR DESCRIPTION
**Pull request by flll**
- Removed explicit values for environment variables in `docker-compose.yml`.
- Utilized default values for better flexibility and maintainability.
- Updated network configuration to use the default bridge driver.

Note: Using `network: default` is sufficient within Docker Compose; there's no need to create a separate `nextcloud-network` for all hosts. 🚀

### CODE REVIEW

**Review by docjyJ — CHANGES_REQUESTED** [review]
It is a good idea to use the default network!

For the variables, maybe you should rename them in the `.env` because there is a small error with the REDIS_HOST_PASWORD.

**Review by Zoey2936 — CHANGES_REQUESTED** [review]
File is created via script: https://github.com/nextcloud/all-in-one/blob/main/manual-install/update-yaml.sh - you need to modify the script not the file

**Comment by flll** [comment]
Thank you for your reviews, Zoey2936, docjyJ, and szaimen.

I've made additional changes to the `update-yaml.sh` script:
- Removed .services[].networks
- Added a special process to remove =%%

Additionally, I've corrected the REDIS_HOST_PASSWORD variable.

Thank you all for your valuable input and contributions.

File: manual-install/update-yaml.sh [hunk 1]
```diff
@@ -41,6 +42,7 @@ sed -i '/AIO_TOKEN/d' containers.yml
 sed -i '/AIO_URL/d' containers.yml
 sed -i '/DOCKER_SOCKET_PROXY_ENABLED/d' containers.yml
 sed -i '/ADDITIONAL_TRUSTED_PROXY/d' containers.yml
+sed -i 's/\(^[[:space:]]*- \([^=]*\)\)=%\2%/\1/' containers.yml
```
Comment (docjyJ) [inline manual-install/update-yaml.sh:45]:
```suggestion
sed -ie 's/\( *- \(\w*\)\)=\${\2\}/\1/' containers.yml
```

I don't really understand the purpose of this line... If it's for environment variables, this regex seems better.

```diff
-      - TURN_SECRET=${TURN_SECRET}
+      - TURN_SECRET
-      - SIGNALING_SECRET=${SIGNALING_SECRET}
+      - SIGNALING_SECRET
-      - ONLYOFFICE_SECRET=${ONLYOFFICE_SECRET}
+      - ONLYOFFICE_SECRET
```

Test online: https://sed.js.org/

**Review by docjyJ — CHANGES_REQUESTED** [review]
Otherwise LGTM

**Review by docjyJ — APPROVED** [review]
LGTM :heart:

**Comment by szaimen** [comment]
Hi @flll thanks a lot for this contribution! I've just invited you to the repo so that collaboration is easier :)

**Comment by szaimen** [comment]
The changes look good in general, but I fear there is another script that needs to be adjusted. See https://github.com/nextcloud/all-in-one/blob/main/nextcloud-aio-helm-chart/update-helm.sh

I've for easier testing pushed your changes to a local branch: https://github.com/nextcloud/all-in-one/pull/5459
After accepting my invite to the repo, you can then go to https://github.com/nextcloud/all-in-one/actions/workflows/update-helm.yml and click the Run workflow button and select the enh/5393/local-branch branch to test your changes. See 
image

**Comment by flll** [comment]
Thank you very much.
I have edited update-helm.yml, pushed it to the enh/5393/local-branch, and ran a workflow against the local branch. I believe it was successful.
I apologize for the trouble I've caused you. Once again, thank you for your assistance.

**Comment by szaimen** [comment]
Merged in https://github.com/nextcloud/all-in-one/pull/5459. Thanks a lot for tackling this and not giving up! 😊

**Pull request closed without merging**